source/_integrations/bluetooth.markdown

```diff
@@ -89,10 +89,43 @@ Deleting the config entry for this integration will release control of the adapt
+
+#### Simple actions that should improve most Bluetooth setups
+
+- Bluetooth adapter hardware:
+  - Buy and use a (good quality) Bluetooth USB adapter based on a newer/modern chip hardware.
+    - If possible consider buying or upgrading to a Bluetooth adapter that has an external antenna. 
+    - While older adapters might work they could have obsolete hardware and/or firmware.
+  - Update to a later released Bluetooth chip firmware on the Bluetooth adapter.
+    - Updating firmware is easy if one is provide by by the manufacturer or the chip maker.
+- Bluetooth devices are RFI sensitive and can be susceptible to all types of EMI/EMF interference:
+  - Use a long USB extension cable to place Bluetooth away from interference and obstacles.
+    - Try to make sure the USB extension cable has proper shielded (thick cables usually do).
+    - Make sure the extension cable has proper shielding and ferrite clamps
+    - Longer USB extension cable also make it easier to orient Bluetooth and antenna.
+  - Only connect the Bluetooth USB adapter to a USB 2.0 ports (or via a powered USB 2.0 hub). 
+    - USB 3.0 ports/peripherals are infamously known to cause EMI/EMF interference.
+  - Shield close unshielded computers and unshielded peripherals by using metal enclosures/chassis.
+    - Single-board-computers and USB hard drives are especially known as source of EMI/EMF.
+  - Try different physical placement and orientations of Bluetooth adapter or its antenna.
+    - Optimal placement of Bluetooth adapter is as much in middle of the house as possible.
+    - Try to place the Bluetooth adapter at some distance way from walls, ceilings and floors.
+    - Try different orientations of the adapters external antenna (or whole Bluetooth adapter).
+
+#### Understanding common root causes of problems in Bluetooth radio setups
+
+- Bluetooth adapter hardware:
+  - Poor placement of the Bluetooth adapter and/or wrong orientation of Bluetooth adapter antenna.
+  - Old, outdated Bluetooth adapter hardware or poor Bluetooth adapter antenna.
+  - Poor or outdated Bluetooth adapter firmware on the Bluetooth adapter.
+-  Bluetooth adapters and Bluetooth devices sensitive to all types of RFI/EMI/EMF interference:
+  - Electromagnetic interference (EMI) from electronics caused by Electromagnetic Fields (EMF).
+  - USB 3.0 ports and computer peripherals are known culprits of EMI/EMF disrupting. Ref. [1](https://www.usb.org/sites/default/files/327216.pdf) and [1](https://www.unit3compliance.co.uk/2-4ghz-intra-system-or-self-platform-interference-demonstration/).
+  - 2.4 GHz Radio Frequency Interference (RFI) from Wi-Fi Routers and Wi-Fi Access Points, or others.
+  - While unsual as Bluetooth is designed to coexist with Wi-Fi, their stronger signal can interfere.
+  - To play it safe best is to try to place your Bluetooth adapter awasy from Wi-Fi access-points.
```

Comment:
Some of this is redundant with the above section. Can you make it a bit more succinct?

Reply:
Do you mean removing the whole bullet point list and instead extend this above section? "**_Most important is to know that other devices that also broadcast on the 2.4 GHz radio band, like USB3 devices (and their cable connections), or other devices are known to affect Bluetooth radio reception. Especially external SSD drives with USB3 cables are known to block the Bluetooth signal. Also, metal casings can decrease the Bluetooth performance of internal Bluetooth Adapters. Recommendation is therefore to at least place USB3 devices and their cables (SSD, etc.) as far away as possible from your Bluetooth adapter, e.g., by using your Bluetooth adapter with a long extension cable that has proper shielding._**"

Reply:
I'm mostly concerned about the duplication between

#### Simple actions that should improve most Bluetooth setups
and 
#### Understanding common root causes of problems in Bluetooth radio setups


The first section has `USB 3.0 ports/peripherals are infamously known to cause EMI/EMF interference.`

the second section has `USB 3.0 ports and computer peripherals are known culprits of EMI/EMF disrupting. Ref. [1](https://www.usb.org/sites/default/files/327216.pdf) and [1](https://www.unit3compliance.co.uk/2-4ghz-intra-system-or-self-platform-interference-demonstration/).`

Reply:
So maybe merge into a single large bullet point list and as such removing the second section like this is better? Like this:

#### Simple actions that should improve most Bluetooth setups and common root causes of interference to be aware of

- Bluetooth adapter hardware:
  - Bad performance from old/outdated Bluetooth adapter hardware or poor Bluetooth adapter antenna.
    - Buy and use a (good quality) Bluetooth USB adapter based on a newer/modern chip hardware.
      - If possible consider buying/upgrading to a Bluetooth adapter that has an external antenna. 
      - While older adapters might work they could have obsolete hardware and/or firmware.
  - Poor or outdated Bluetooth adapter firmware on the Bluetooth adapter.
    - Update to a later released Bluetooth chip firmware on the Bluetooth adapter.
      - Updating firmware is usually easy if one is provide by the manufacturer or the chip maker.
- Bluetooth adapters are RFI sensitive and can be very susceptible to all types of EMI/EMF interference:
  - Poor placement of the Bluetooth adapter and/or wrong orientation of Bluetooth adapter antenna.
    - Use a long USB extension cable to place Bluetooth adapter away from interference and obstacles.
      - Make sure the USB extension cable has proper shielded (thick cables usually have this).
        - Preferably by a longer extension cable specified to have proper shielding and ferrite clamps.
        - Using a USB extension cable also make it easier to orient the Bluetooth adapter/antenna.
    - Try different physical placement and orientations of Bluetooth adapter or its antenna.
      - Optimal placement of Bluetooth adapter is as much in middle of the house as possible.
      - Try to place the Bluetooth adapter at some distance way from walls, ceilings and floors.
      - Try different orientations of the adapters external antenna (or whole Bluetooth adapter).
  - USB 3.0 ports/computers/peripherals are known culprits of RFI/EMI/EMF disruption. (See Ref. [1](https://www.usb.org/sites/default/files/327216.pdf) and [2](https://www.unit3compliance.co.uk/2-4ghz-intra-system-or-self-platform-interference-demonstration/)).
    - Make sure to only connect the Bluetooth USB adapter to a USB 2.0 port (and not to a USB 3.x port). 
      - If your computer only has a USB 3.x port then connect the adapter via a powered USB 2.0 hub.
        - A USB 2.0 hub will convert USB 3.0 to a USB 2.0 port.
        - Buy a USB 2.0 hub that has an external power supply to ensure fulfil power requirements.
    - Shield close unshielded computers and unshielded peripherals by using metal enclosures/chassis.
      - Single-board-computers and USB 3.x hard drives are especially known as source of EMI/EMF.
      - Also be sure to use properly shielded USB cables for any such peripherals/devices too.
  - 2.4 GHz RF Interference (RFI) from Wi-Fi Routers and Wi-Fi Access Points, or other type of devices.
    - While unusual as Bluetooth is designed to coexist with Wi-Fi, their stronger signal can interfere.
      - To play it safe best to try to place your Bluetooth adapter away from Wi-Fi access-points.
    - Place away from electrical/power wires/cables and power supplies and even household appliances.

Reply:
Yes thats better.

Reply:
OK thanks for the feedback, updated now.